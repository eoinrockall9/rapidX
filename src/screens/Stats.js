import React, { Component } from 'react';
import { Container, Header, Content, Tab, Tabs, Text } from 'native-base';


export default class TabsExample extends Component {
  render() {
    return (
      <Container>
        <Tabs>
          <Tab heading="Daily">
            <Text>Daily</Text>
          </Tab>
          <Tab heading="Weekly">
            <Text>Weekly</Text>
          </Tab>
          <Tab heading="Monthly">
            <Text>Monthly</Text>
          </Tab>
        </Tabs>
      </Container>
    );
  }
}