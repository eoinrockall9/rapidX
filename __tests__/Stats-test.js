import React from 'react';
import Stats from '../src/screens/Stats';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
    const tree = renderer.create(<Stats />).toJSON();
    expect(tree).toMatchSnapshot();
})