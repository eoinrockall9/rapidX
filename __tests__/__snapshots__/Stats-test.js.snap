// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff",
      "flex": 1,
      "height": 1334,
    }
  }
>
  <View
    onLayout={[Function]}
    style={
      Array [
        Object {
          "flex": 1,
        },
        undefined,
      ]
    }
  >
    <View
      style={
        Array [
          Object {
            "borderBottomWidth": 0.5,
            "borderColor": "#a7a6ab",
            "elevation": 3,
            "flexDirection": "row",
            "height": 50,
            "justifyContent": "space-around",
            "shadowColor": undefined,
            "shadowOffset": undefined,
            "shadowOpacity": undefined,
            "shadowRadius": undefined,
          },
          Object {
            "backgroundColor": "#F8F8F8",
          },
          Object {},
        ]
      }
    >
      <View
        accessible={true}
        focusable={true}
        isTVSelectable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "flex": 1,
            "opacity": 1,
          }
        }
      >
        <View
          active={true}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#F8F8F8",
              "flex": 1,
              "flexDirection": "row",
              "justifyContent": "center",
            }
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#007aff",
                  "fontFamily": "System",
                  "fontSize": 16,
                  "fontWeight": "600",
                  "marginLeft": 7,
                  "marginRight": 7,
                },
                Object {
                  "fontSize": 15,
                },
                undefined,
                Object {
                  "color": "#007aff",
                },
              ]
            }
            uppercase={false}
          >
            Daily
          </Text>
        </View>
      </View>
      <View
        accessible={true}
        focusable={true}
        isTVSelectable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "flex": 1,
            "opacity": 1,
          }
        }
      >
        <View
          active={false}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#F8F8F8",
              "flex": 1,
              "flexDirection": "row",
              "justifyContent": "center",
            }
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#6b6b6b",
                  "fontFamily": "System",
                  "fontSize": 16,
                  "marginLeft": 7,
                  "marginRight": 7,
                },
                Object {
                  "fontSize": 15,
                },
                undefined,
                Object {
                  "color": "#6b6b6b",
                },
              ]
            }
            uppercase={false}
          >
            Weekly
          </Text>
        </View>
      </View>
      <View
        accessible={true}
        focusable={true}
        isTVSelectable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "flex": 1,
            "opacity": 1,
          }
        }
      >
        <View
          active={false}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#F8F8F8",
              "flex": 1,
              "flexDirection": "row",
              "justifyContent": "center",
            }
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#6b6b6b",
                  "fontFamily": "System",
                  "fontSize": 16,
                  "marginLeft": 7,
                  "marginRight": 7,
                },
                Object {
                  "fontSize": 15,
                },
                undefined,
                Object {
                  "color": "#6b6b6b",
                },
              ]
            }
            uppercase={false}
          >
            Monthly
          </Text>
        </View>
      </View>
      <View
        style={
          Object {
            "backgroundColor": "#007aff",
            "bottom": 0,
            "height": 4,
            "left": 0,
            "position": "absolute",
            "width": 250,
          }
        }
      />
    </View>
    <RCTScrollView
      alwaysBounceVertical={false}
      automaticallyAdjustContentInsets={false}
      contentOffset={
        Object {
          "x": 0,
        }
      }
      directionalLockEnabled={true}
      horizontal={true}
      keyboardDismissMode="on-drag"
      keyboardShouldPersistTaps="handled"
      onMomentumScrollBegin={[Function]}
      onMomentumScrollEnd={[Function]}
      onScroll={[Function]}
      pagingEnabled={true}
      scrollEnabled={true}
      scrollEventThrottle={16}
      scrollsToTop={false}
      showsHorizontalScrollIndicator={false}
    >
      <View>
        <View
          style={
            Object {
              "width": 750,
            }
          }
        >
          <View
            heading="Daily"
            style={
              Object {
                "backgroundColor": "#FFF",
                "flex": 1,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#000",
                  "fontFamily": "System",
                  "fontSize": 16,
                }
              }
              uppercase={false}
            >
              Daily
            </Text>
          </View>
        </View>
        <View
          style={
            Object {
              "width": 750,
            }
          }
        >
          <View
            heading="Weekly"
          />
        </View>
        <View
          style={
            Object {
              "width": 750,
            }
          }
        >
          <View
            heading="Monthly"
          />
        </View>
      </View>
    </RCTScrollView>
  </View>
</View>
`;
